docs/docs/installation/kustomize.md

```diff
@@ -76,7 +76,7 @@ kind: Kustomization
 
 # update the version to the release you need
 resources:
-  - https://github.com/grafana/grafana-operator/releases/download/v5.0.10/kustomize-cluster_scoped.yaml
+  - https://github.com/grafana/grafana-operator/releases/download/v5.12.0/kustomize-cluster_scoped.yaml
```

Comment:
We can use a parameter here so it will always be up to date
```suggestion
  - https://github.com/grafana/grafana-operator/releases/download/{{<param version>}}/kustomize-cluster_scoped.yaml
```